### DISCUSSION TITLE
Simpler QStepper - Without content

### DISCUSSION THREAD

**Discussion by 549393092** [quasarframework/quasar#16654] [General - Components / Directives / etc] [answered]
Dear Quasar Team,

I did a demo to explain this, could you please have a look?
https://codepen.io/Stanley-549393092/pen/LYqXzQo?editors=1010
I just want to have a simple stepper without content, however, the empty space still occupy the page.
How to hide it, thanks!

**Comment by metalsadman** [comment, marked as answer]
```css
.q-stepper__content{
  display: none;
}
```

**Reply by 549393092** [reply]
Thank you! It works.

**Comment by dhantoot** [comment]
<q-step
      . . .
      :class="{
         'hidden': true
      }">
</q-step>
